README.md

```diff
@@ -28,3 +33,23 @@ See [Code of Conduct](CODE_OF_CONDUCT.md).
 
 This software was developed by the National Snow and Ice Data Center with funding from
 multiple sources.
+
+
+## Development
+
+To develop the workshop website locally, install the dependencies via `conda`:
+
+```
+$ conda env create
+$ conda activate qgreenland-researcher-workshop
+```
+
+Then use the `quarto preview` command to start a local server for the website
+that will live-reload on changes to source content (`.qmd` files).
+
+
+### Fixing spelling mistakes
+
+`mdspell` is used to check for spelling pistakes in source files. If the linter
+catches spelling mistakes, use `./scripts/spell-check.sh -i` to interactively
+correct mistakes (or add exceptions).
```

Comment:
This is not covered in the contributing doc, so if you agree we should keep that file, we should migrate this over there.

Reply:
Yep, I'll transfer this info.